examples/chain/21_MsgRevoke.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"fmt"
+	"github.com/InjectiveLabs/sdk-go/client/common"
+	chainclient "github.com/InjectiveLabs/sdk-go/client/chain"
+	rpchttp "github.com/tendermint/tendermint/rpc/client/http"
+	authztypes "github.com/cosmos/cosmos-sdk/x/authz"
+	"time"
+)
+
+func main() {
+	network := common.LoadNetwork("testnet", "k8s")
+	tmRPC, err := rpchttp.New(network.TmEndpoint, "/websocket")
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	senderAddress, cosmosKeyring, err := chainclient.InitCosmosKeyring(
+		"/Users/akalantzis/.injectived",
```

Comment:
we shouldn't commit this of course @achilleas-kal

Reply:
At minimum $HOME/.injectived but ideally also user configurable, with default being set to $HOME/.injectived if unconfigured

Reply:
It's a dummy keyring/private key like we have in sdk-python, I think it makes the examples more clear @albertchon

Reply:
Sure I'll change to $HOME/.injectived